test/experiments/calibrations/grid_search/corner_plot.py

```diff
@@ -0,0 +1,179 @@
+# -*- coding: utf-8 -*-
+"""
+Spyder Editor
+
+This is a temporary script file.
+"""
+
+import numpy as np
+import netCDF4 as nc
+import matplotlib.pyplot as plt
+import argparse
+import json
+import matplotlib.ticker as mticker
+import matplotlib as mpl
+
+###
+
+# This code is designed to receive loss.nc from CalibrateEDMF's loss_map.jl function
```

Comment:
CalibrateEDMF?

Reply:
Great catch. This tool is adapted from a similar tool in CalibrateEDMF.